### PR TITLE
www-apps/cgisysinfo: update SRC_URI, HOMEPAGE, REPO_URI

### PR DESCRIPTION
**Pull request by wimmuskee**
Closes: https://bugs.gentoo.org/626300

Apparently, the Github 0.4.1 version requires an eautoreconf.
The Github compiled bin has the same md5sum as the one from the tree.

### CODE REVIEW

File: www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
+inherit autotools
+
 if [[ ${PV} = *9999* ]]; then
 	WANT_AUTOMAKE="1.10"
-	inherit autotools mercurial
-	EHG_REPO_URI="http://hg.rafaelmartins.eng.br/cgisysinfo/"
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/rafaelmartins/${PN}"
 	KEYWORDS=""
+	DOCS="README.md AUTHORS NEWS"
```
Comment (monsieurp) [inline www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild]: Could you define `DOCS` as an array?

File: www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild [hunk 2]
```diff
@@ -3,18 +3,22 @@
...
 else
-	SRC_URI="http://distfiles.rafaelmartins.eng.br/${PN}/${P}.tar.bz2"
+	SRC_URI="https://github.com/rafaelmartins/${PN}/archive/${PV}.tar.gz"
 	KEYWORDS="~amd64 ~x86"
+	DOCS="README AUTHORS NEWS"
```
Comment (monsieurp) [inline www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild]: Ditto.

**Comment by rafaelmartins** [comment]
@wimmuskee the md5sums are definitely different. the file in distfiles.gentoo.org was generated with `make dist`, the github archive is just a git archive, without prebuilt autotools scripts. the must differ. :)

File: www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild [hunk 3]
```diff
@@ -3,18 +3,22 @@
...
 	WANT_AUTOMAKE="1.10"
-	inherit autotools mercurial
-	EHG_REPO_URI="http://hg.rafaelmartins.eng.br/cgisysinfo/"
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/rafaelmartins/${PN}"
 	KEYWORDS=""
+	DOCS="README.md AUTHORS NEWS"
 else
-	SRC_URI="http://distfiles.rafaelmartins.eng.br/${PN}/${P}.tar.bz2"
+	SRC_URI="https://github.com/rafaelmartins/${PN}/archive/${PV}.tar.gz"
```
Comment (doughdemon) [inline www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild]: `-> ${P}.tar.gz` is missing

Comment (wimmuskee) [reply www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild]: good call, thx

File: www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild (hunk 2 above)
Comment (wimmuskee) [reply www-apps/cgisysinfo/cgisysinfo-0.4.1.ebuild]: both fixed

**Comment by wimmuskee** [comment]
fixed change requests and rebased

**Comment by wimmuskee** [comment]
@rafaelmartins since you are the maintainer, can you approve and merge this PR please?

**Comment by flappyports** [comment]
no activity

**Comment by rafaelmartins** [comment]
pushed. thanks @wimmuskee and sorry for the delay

**Pull request closed without merging**